DCO

```diff
@@ -2,8 +2,9 @@ Developer Certificate of Origin
 Version 1.1
 
 Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
-660 York Street, Suite 102,
-San Francisco, CA 94110 USA
+1 Letterman Drive
+Suite D4700
```

Comment:
Does this line need the building number too? `Building D, Suite D4700`? as in https://www.linuxfoundation.org/about/contact/